### ISSUE TITLE
Google Webmaster

### ISSUE THREAD

**Issue by artlogic** [arsmentis/arsmentis#21]
Setup google webmaster tools for the new site.

**Comment by artlogic** [comment]
@SKBruck - please go in and review the settings. I've granted you access. Do anything that makes sense.

**Comment by SKBruck** [comment]
I spent some time researching how to use the webmaster tools, and I made one change that was suggested often, which was to use the data highlighter to highlight your company name, address, and phone number. Using structured data was also suggested, but I need to look into that more because that uses html. Other than that, it seems that the tools are set up the way they should be.

It might be good to go back over the tool settings after launching phase 2, so instead of closing this issue, I'm re-labelling as phase 2 so I don't forget. Let me know if you'd prefer I close it.
